Formula/ts-node.rb

```diff
@@ -0,0 +1,29 @@
+require "language/node"
+
+class TsNode < Formula
+  desc "TypeScript execution and REPL for node.js"
+  homepage "https://github.com/TypeStrong/ts-node"
+  url "https://registry.npmjs.org/ts-node/ts-node-8.0.2.tgz"
+  sha256 "ba6a690ddf1e441425fe47d37b06e61fe4a8d3be10430b17eb286630d9569324"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+    # Adding typescript to library's libexec
```

Comment:
I don't understand why you need to do this. Shouldn't `npm install` install `typescript` already ?

Reply:
I did this as I found that both `brew install typescript` and `npm install typescript -g` did not work for `ts-node`. I'm asking for help for any cases like this (that is, how to resolve external packages via `npm install -g`). Also, my humble understanding on how Node.js, npm and CommonJS modules work out seems not enough.

Another observation from `package.json` is that `typescript` is (perhaps intentionally) listed in `peerDependencies ` and `devDependencies`.

TypeStrong/ts-node#765
https://github.com/TypeStrong/ts-node/blob/master/package.json#L67-L70